ui/app/services/token.js

```diff
@@ -49,10 +49,9 @@ export default class TokenService extends Service {
     this.secret = token.secret;
   }
 
-  @computed('secret', 'fetchSelfToken.lastSuccessful.value')
+  @computed('fetchSelfToken.lastSuccessful.value')
   get selfToken() {
-    if (this.secret) return this.get('fetchSelfToken.lastSuccessful.value');
-    return undefined;
+    return this.get('fetchSelfToken.lastSuccessful.value');
```

Comment:
Minor nitpick, but I think the "reads" helper is slightly more idiomatic - https://api.emberjs.com/ember/3.27/classes/@ember%2Fobject%2Fcomputed/methods/alias?anchor=reads

Reply:
Make sense. I updated the PR, ran tests and tested locally: all good!

Reply:
Quick question: would it be better to use `@alias` versus `@reads`?

Reply:
Oh yeah - I think you're right, sorry.

If the user is just reading this value, then it shouldn't matter, but to match the old logic, I think `alias` is more accurate.

Appreciate the correction :)